### PR TITLE
BeSman repo updated with 'bes open' command

### PR DESCRIPTION
**Pull request by jobyko**
1.  Added 'bes open'  command.
2.  Removed 'bes publish' command.

### CODE REVIEW

File: src/main/bash/besman-repo-helpers.sh [hunk 1]
```diff
@@ -57,4 +57,19 @@ function __besman_check_github_id
     # __besman_error_rollback "$environment"
     return 1
   fi
-}
\ No newline at end of file
+}
+
+
+function __besman_open_file
```
Comment (asa1997) [inline src/main/bash/besman-repo-helpers.sh:63]: This function is already available in env helper file https://github.com/Be-Secure/BeSman/blob/master/src/main/bash/besman-env-helpers.sh#L101

**Comment by asa1997** [comment]
@jobyko @panickervinod LGTM.

@jobyko Please address the comments.

**Review by asa1997 — APPROVED** [review]

File: src/main/bash/besman-repo-helpers.sh (hunk 1 above)
Comment (jobyko) [reply src/main/bash/besman-repo-helpers.sh]: Removed repeated function, function __besman_open_file from repo helper file.

**Comment by jobyko** [comment]
Removed repeated function, __besman_open_file from repo helper file.

**Comment by panickervinod** [comment]
@jobyko @asa1997

**Pull request closed without merging**